src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-submitter.html

```diff
@@ -1,5 +1,5 @@
 <div>
     User IDs and/or <em>external</em> group names of person or people permitted to respond to the input, separated by ','.
     Spaces will be trimmed. This means that "alice, bob, blah " is the same as "alice,bob,blah".<br>
-    <strong>Note: Jenkins administrators are able to respond to the input regardless of the value of this parameter.</strong>
+    <strong>Note: Jenkins administrators are able to respond to the input regardless of the value of this parameter. And users with cancel permissions may also respond with 'Abort' to the input.</strong>
```

Comment:
A link to the page that describes permissions may help the reader understand.
```suggestion
    <strong>Note: Jenkins administrators are able to respond to the input regardless of the value of this parameter. And users with <a href="https://www.jenkins.io/doc/book/security/access-control/permissions/#job-permissions">Job/cancel permission</a> may also respond with 'Abort' to the input.</strong>
```

Reply:
That will be really useful.